third-party/java/intellij/BUCK

```diff
@@ -48,3 +49,8 @@ prebuilt_jar(
     name = "jdom",
     binary_jar = "jdom.jar",
 )
+
+prebuilt_jar(
+    name = "android",
+    binary_jar = "android.jar",
```

Comment:
yes, it's from an IntelliJ installation. it does not need to be a specific version.

@styurin which piece of that do you want in a separate PR?